### PR TITLE
Provide cli information for baseline.xml

### PR DESCRIPTION
**Pull request by madlymad**
Documentation about creation baseline file via cli is missing and is misleading.

Attached a mirror suggestion for improvement!

### CODE REVIEW

File: website/docs/introduction/baseline.md [hunk 1]
```diff
@@ -29,6 +29,12 @@ the `--report txt:path/to/report` cli flag.
 </SmellBaseline>
 ```
 
+### CLI
+To to generate yourself a `baseline.xml` you need to provide the same config as the the rules you are going to scan your project.
+```shell
+java -jar detekt-cli-all.jar --plugins detekt-formatting.jar --build-upon-default-config --config my-detekt-config.yml --baseline path/to/new/baseline.xml --create-baseline
```
Comment (cortinico) [inline website/docs/introduction/baseline.md:35]: Can we put those on separate lines?

Comment (madlymad) [reply website/docs/introduction/baseline.md]: 👍 Done!

**Review by BraisGabin — APPROVED** [review]

**Comment by 3flex** [comment]
I think this should be simplified - all it needs to say is that the --create-baseline parameter should be added to the command line to create a baseline.

**Comment by 3flex** [comment]
Sorry, yes you're right, you need to pass a baseline file output path as well.

My point is that if you have a detekt command, and you want to generate a baseline, all you need to do is append the baseline parameters to the command.

**Comment by madlymad** [comment]
@3flex thanks for explaining that and not that I work with detekt for more 3 weeks this seems pretty obvious. When I start working with it 3 weeks ago this seemed to as a missing/misleading documentation.

File: website/docs/introduction/baseline.md [hunk 2]
```diff
@@ -29,6 +29,18 @@ the `--report txt:path/to/report` cli flag.
...
+To generate yourself a `baseline.xml` you need to provide the same config as the the rules you are going to scan your project.
+
+```sh
+java -jar detekt-cli-all.jar \
+  --plugins detekt-formatting.jar \
+  --build-upon-default-config \
+  --config path/to/config/detekt/detekt.yml \
+  --baseline path/to/new/config/detekt/baseline.xml \
+  --create-baseline
+```
```
Comment (cortinico) [inline website/docs/introduction/baseline.md:42]:
```suggestion
```diff
java -jar detekt-cli-all.jar \
  --plugins detekt-formatting.jar \
  --build-upon-default-config \
  --config path/to/config/detekt/detekt.yml \
+ --baseline path/to/new/config/detekt/baseline.xml \
+ --create-baseline
```
```

File: website/versioned_docs/version-1.23.5/introduction/baseline.md [hunk 3]
```diff
@@ -29,6 +29,18 @@ the `--report txt:path/to/report` cli flag.
...
+To generate yourself a `baseline.xml` you need to provide the same config as the the rules you are going to scan your project.
+
+```sh
+java -jar detekt-cli-all.jar \
+  --plugins detekt-formatting.jar \
+  --build-upon-default-config \
+  --config path/to/config/detekt/detekt.yml \
+  --baseline path/to/new/config/detekt/baseline.xml \
+  --create-baseline
+```
```
Comment (cortinico) [inline website/versioned_docs/version-1.23.5/introduction/baseline.md:42]:
```suggestion
```diff
java -jar detekt-cli-all.jar \
  --plugins detekt-formatting.jar \
  --build-upon-default-config \
  --config path/to/config/detekt/detekt.yml \
+ --baseline path/to/new/config/detekt/baseline.xml \
+ --create-baseline
```
```

**Review by cortinico — APPROVED** [review]

**Pull request merged**